communications/README.md

```diff
@@ -0,0 +1,11 @@
+# Communications
+
+## Async written first
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
+- If you want to discuss a process change, consider the smallest reasonable change and create a GitHub PR in this repo as a proposal. @-mention and assign specific individuals for review.
+- Only when you have exhausted discussions in Clubhouse and/or GitHub, consider scheduling a meeting with a clear agenda to determine next steps.
+- When using Slack, prefer using public channels over DMs. @-mention specific individuals with a question if you want a response. Prefer using threads.
+- Slack is an async tool. Don't expect people to respond immediately. Don't feel pressured to respond immediately.
+- Prefer Clubhouse over Slack, and Slack over email.
```

Comment:
I might qualify this as "For product discussion, ..."

Reply:
I don't want to qualify it, because I think it applies more broadly. So I'd rather just delete it then. Will do that. Especially since the previous sentences already explain how to use Clubhouse.